vm/adb/adb.go

```diff
@@ -153,6 +153,14 @@ func findConsole(adb, dev string) string {
 }
 
 func findConsoleImpl(adb, dev string) (string, error) {
+	// Attempt to find an exact match, at /dev/ttyUSB.{SERIAL}
+	// This is something that can be set up on Linux via 'udev' rules
+	exactCon := "/dev/ttyUSB." + dev
+	if _, err := osutil.IsExist(exactCon) {
```

Comment:
This doesn't quite build...
https://travis-ci.org/google/syzkaller/jobs/407355621

Reply:
That's what I get for trying to quickly edit something and force-push.  I've fixed this, CI looks green